lib/ui/painting.dart

```diff
@@ -1316,6 +1316,26 @@ Future<Null> _decodeImageFromListAsync(Uint8List list, ImageDecoderCallback call
   callback(frameInfo.image);
 }
 
+/// The encoding formats supported by the [encodeImage].
+enum EncodedImageFormat {
+  // Be conservative with the formats we expose. We can increase the list in future but reducing
+  // it is more difficult.
+  JPEG,
+  PNG,
+  WEBP,
+}
+
+/// Callback signature for [encodeImage].
+typedef void ImageEncoderCallback(Uint8List result);
+
+/// Convert an [Image] object into a byte array/
+///
+/// [format] is encoding format to be used.
+///
+/// [quality] is a value in [0, 100] where 0 corresponds to the lowest quality.
```

Comment:
Please start each paragraph with a capital letter, e.g. by saying `The [format] argument is...`

Reply:
Rather than `in [0, 100]` let's say something like `in the range 0 to 100`.

Reply:
Done. Improved the wording and provide more information of what lowest and highest quality mean.